postgres-driver/application.go

```diff
@@ -160,6 +151,211 @@ func (a *dbApplication) toApplication() *repository.Application {
 	}
 }
 
+type insertDBApp struct {
+	ApplicationID string         `db:"application_id"`
+	UserID        sql.NullString `db:"user_id"`
+	Name          sql.NullString `db:"name"`
+	ContactEmail  sql.NullString `db:"contact_email"`
+	Description   sql.NullString `db:"description"`
+	Owner         sql.NullString `db:"owner"`
+	URL           sql.NullString `db:"url"`
+	CreatedAt     time.Time      `db:"created_at"`
+	UpdatedAt     time.Time      `db:"updated_at"`
+}
+
+func extractInsertDBApp(app *repository.Application) *insertDBApp {
+	return &insertDBApp{
+		ApplicationID: app.ID,
+		UserID:        newSQLNullString(app.UserID),
+		Name:          newSQLNullString(app.Name),
+		ContactEmail:  newSQLNullString(app.ContactEmail),
+		Description:   newSQLNullString(app.Description),
+		Owner:         newSQLNullString(app.Owner),
+		URL:           newSQLNullString(app.URL),
+		CreatedAt:     app.CreatedAt,
+		UpdatedAt:     app.UpdatedAt,
+	}
```

Comment:
There's a number of Application fields that the PUB sends for creating an Application that are missing here. Can you confirm if you should add them or if they are not needed?

- Dummy (bool)
- Free Tier (bool)
- FreeTierApplicationAccount (repository.FreeTierApplicationAccount)
- Status (string)

Reply:
I want to add those fields in other PR, that also correctly adds them to select statement.